resources/js/app.js

```diff
@@ -55,6 +55,32 @@ import VueApollo from "vue-apollo";
 import VueClipboard from "vue-clipboard2";
 import Vuelidate from "vuelidate";
 import VueRouter from "vue-router";
+import VueMarkdownEditor from '@kangc/v-md-editor';
+import '@kangc/v-md-editor/lib/style/base-editor.css';
+import vuepressTheme from '@kangc/v-md-editor/lib/theme/vuepress.js';
+import '@kangc/v-md-editor/lib/theme/style/vuepress.css';
+import enUS from '@kangc/v-md-editor/lib/lang/en-US';
+import VMdPreview from '@kangc/v-md-editor/lib/preview';
+import '@kangc/v-md-editor/lib/style/preview.css';
+import githubTheme from '@kangc/v-md-editor/lib/theme/github.js';
+import '@kangc/v-md-editor/lib/theme/style/github.css';
+
+
+
+
+// Prism
+import Prism from 'prismjs';
+// highlight code
+import 'prismjs/components/prism-json';
+
+VueMarkdownEditor.use(vuepressTheme, {
+  Prism,
+});
+VueMarkdownEditor.lang.use('en-US', enUS);
+
+Vue.use(VueMarkdownEditor);
+VMdPreview.use(githubTheme);
+Vue.use(VMdPreview);
```

Comment:
We could probably extract this into a component, but we can do that later I suppose unless you are down to give it a shot

Reply:
I've never done that before in the context of the `app.js` file. I would prefer to punt on that one until I can get a better understanding